README.md

```diff
@@ -101,10 +110,35 @@ You can also deploy to all customers for a given environment by running e.g. `ca
 
 Using branches for stages, i.e. demo and production branches, can clutter up your branches screen. This can be particularly awkward when running CI and keeping track of multiple active branches. To resolve this you can optionally configure epi_deploy to use tags for this instead of branches.
 
-1. Update to epi_deploy 2.2.0 or greater.
-2. Add config/epi_deploy.rb to your application.
-3. Add `EpiDeploy.use_tags_for_deploy = true` to the newly created config file.
-4. Delete your stage branches on gitlab (likely `production`, `demo`, `qa`).
-5. Push your changes and deploy to a demo site to test it is working correctly.
+Tags will be automatically created for each successful deployment with the format `environment.stage-timestamp`, for example `production.epigenesys-2024_10_03-12_20_09`, and pushed to the remote.
```

Comment:
I think this is slightly different to what I assumed we'd get - I thought we'd have a `production.epigenesys` tag which changes on every re-deploy (which could be alongside the timestamped version). It's going to make things more difficult to parse in epiApps if we need to grab all `production.{customer}-{timestamp}` tags then figure out which one is the most recent one...

Reply:
Hi James, part of this work was to try and avoid moving the deployment tags like we trialled with F4X, because this wasn't very reliable and according to the documentation for `git tag` moving a tag is not recommended when the tag has already been pushed: <https://git-scm.com/docs/git-tag#_on_re_tagging>. The tags produced for each deployment stage can be sorted lexicographically to find the newest tag, as the timestamp is in the format `yyyy_mm_dd-hh_mm_ss`